Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:3.8.6-alpine3.12 AS base
```

Comment:
While usually `alpine` based images are very good, for python it seems it isn't the best choice. See https://pythonspeed.com/articles/alpine-docker-python/ and https://pythonspeed.com/articles/base-image-python-docker-images/

Reply:
Yeah, I actually had some musl problems before, but never ran into long build time issues. But I think I'll switch to debian-slim.